### PR TITLE
[SLIDES]: migration - model - OT102-30

### PR DESCRIPTION
**Pull request by maxicuyo94**
# [OT102-30](https://alkemy-labs.atlassian.net/browse/OT102-30)

Description
AS: admin user
I WANT: to manage images
TO: efficiently modify visual content

Acceptance criteria: 
The slides on the home screen will be dynamically managed by the site administrator. They will have as fields imageUrl, text, order and organizationId (as they will belong to an ngo).

### CODE REVIEW

File: migrations/20211123134047-create-slide.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      imageUrl: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        unique: true,
+      },
+      text: {
+        type: Sequelize.STRING,
```
Comment (dvelezroman) [inline migrations/20211123134047-create-slide.js]: deberia ser mas bien de tipo TEXT

File: models/slide.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      },
+      text: {
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+      order: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+      },
+      // organizationId: {
```
Comment (dvelezroman) [inline models/slide.js]: si no se usa .. borrar

File: models/slide.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      //   type: DataTypes.INTEGER,
+      //   allowNull: false,
+      // }
+    },
+    {
+      sequelize,
+      modelName: 'Slide',
```
Comment (dvelezroman) [inline models/slide.js:39]: completar con las configuraciones que coloque en otrs PRs ... en el #3

**Review by dvelezroman — CHANGES_REQUESTED** [review]
Buen Pr... corregir y arreglar el PR la descripcion y algo del codigo del model

File: migrations/20211123134047-create-slide.js (hunk 1 above)
Comment (maxicuyo94) [reply migrations/20211123134047-create-slide.js]: cambiado

File: models/slide.js (hunk 2 above)
Comment (maxicuyo94) [reply models/slide.js]: eliminado

File: models/slide.js (hunk 3 above)
Comment (maxicuyo94) [reply models/slide.js:39]: agregado createdAt updatedAt deletedAt y paranoid

**Review by JonatanGarbuyo — APPROVED** [review]

**Review by dvelezroman — APPROVED** [review]
Excelent

**Review by gabodmc — APPROVED** [review]

**Pull request merged**